WISE-Community/WISE
# Clarify difference between sharing runs and units

To test:
- Verify that share run option in run menu now reads "Share Access" (instead of "Share") and has new icon
- Verify that share run dialog title is now "Share Classroom Access" (instead of "Share Classroom Unit")
- Review new text in share run dialog that explains functionality and ensure that sharing runs still works
- Verify that you can copy a unit by clicking "create a copy" link text in share run dialog
- Verify that copying a unit from unit menu (in the unit libraries) still works properly
- Review new text in share unit dialog that explains functionality and ensure that sharing units still works

Closes #2787 .
